features/support/api_helpers.rb

```diff
@@ -52,15 +56,15 @@ def ensure_user_exists(user_details)
   creation_response = call_api(:post, "/users", payload: {
     users: user_details,
     updated_by: "functional tests",
-  })
+  }, safe_for_smoke_tests: true)
```

Comment:
I guess the thing to highlight is that this method is only used by `I have a production ... user` step definition and probably shouldn't be used by anything else. Most other steps just create a new random user.